src/components/app-top-bar.js

```diff
@@ -75,6 +76,10 @@ const styles = {
         margin: theme.spacing(1.5),
         fontWeight: 'bold',
     }),
+    divRunButton: {
+        marginRight: '100px',
```

Comment:
That would be better with flexible design instead of hard values. With a smaller window, there is a big gap where space could be saved.

Reply:
Done